universum/configuration_support.py

```diff
@@ -8,14 +8,133 @@
     "Step",
     "Configuration",
     "Variations",
+    "combine",
     "set_project_root",
     "get_project_root"
 ]
 
 
+# pylint: disable-msg=line-too-long
```

Comment:
Maybe move this to the beginning of the file, as we disable this rule for the whole module.

Reply:
or I can enable it back after the class definition. Which one is preferable?

Reply:
In this particular case I'd choose disabling for the whole file, so if needed we can add more description to `Configuration` as well.